src/components/WorkMetadata/WorkMetadata.tsx

```diff
@@ -424,20 +450,35 @@ const WorkMetadata: React.FunctionComponent<Props> = ({
         <a href={handleUrl}>
           <FontAwesomeIcon icon={faExternalLinkAlt} size="sm" /> {handleUrl}
         </a>
+        <span className="actions with-label">
+          {showDownloadLink && (
```

Comment:
Maybe the test should be above the span, no need to output the span with nothing in it.

Reply:
I agree and made that change.